sdk/trace/tracetest/example_test.go

```diff
@@ -0,0 +1,123 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package tracetest_test
+
+import (
+	"context"
+	"fmt"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/sdk/trace/tracetest"
+)
+
+
+func processItem(ctx context.Context, itemID string) {
+	tracer := otel.Tracer("example/processor")
+	ctx, span := tracer.Start(ctx, "processItem")
+	defer span.End()
+	
+	span.SetAttributes(attribute.String("item.id", itemID))
+	// Simulate some processing work
+	span.AddEvent("processing started")
+}
+
+
+func Example() {
```

Comment:
How is this example different from the one below?

Reply:
Right, but that provides a span creation/processing example. There are no differences related to tracetest.